src/Components/CreateImageWizard/WizardStepReview.js

```diff
@@ -107,7 +107,7 @@ class WizardStepReview extends Component {
                 <Title headingLevel="h2" size="xl">Review</Title>
                 <TextContent>
                     <Text component={ TextVariants.small }>
-                    Review the information and click Create image
+                    Review the information and click Create botton
```

Comment:
I think it would be best if this said `Review the information and click the Create button`

Reply:
Thank you so much @jkozol !

Reply:
Updated.